app/lib/presentation/organisms/select_language_card_organism.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:word_link/domain/controllers/controllers.dart';
+import 'package:word_link/presentation/core/global_variables.dart';
+
+class SelectLanguageCardOrganism extends StatelessWidget {
+  const SelectLanguageCardOrganism({
+    required this.language,
+    required this.isSelected,
+    required this.onTap,
+    super.key,
+  });
+
+  final LanguageEnum language;
+  final bool isSelected;
+  final VoidCallback onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: onTap,
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(GlobalVariables.defaultRadius),
+          boxShadow: [
+            BoxShadow(
+              color: Theme.of(context).colorScheme.shadow,
+              blurRadius: 10,
+              offset: const Offset(0, 5),
+            ),
+          ],
+          border: isSelected
+              ? Border.all(
+                  color: Theme.of(context).colorScheme.primary, width: 3)
+              : null,
+        ),
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(GlobalVariables.defaultRadius),
+          child: Stack(
+            fit: StackFit.expand,
+            children: [
+              Image.network(
+                language.flagUrl,
+                fit: BoxFit.cover,
+                errorBuilder: (context, error, stackTrace) => Container(
+                  color: Theme.of(context).colorScheme.primaryContainer,
+                ),
+              ),
+              Container(
+                decoration: BoxDecoration(
+                  gradient: LinearGradient(
+                    begin: Alignment.topCenter,
+                    end: Alignment.bottomCenter,
+                    colors: [
+                      Colors.transparent,
+                      Theme.of(context).colorScheme.scrim,
```

Comment:
We are using `Theme.of(context).colorScheme` several times in the page.
Move it to a variable